Is UTC stripped out too? Should we update the Readme?

I'm using the env var `$env:CHRONO_TZ_TIMEZONE_FILTER='(Europe/Berlin)'` and building with `cargo build`, then I get this error:

```
Compiling chrono-tz v0.8.6
error[E0599]: no variant or associated item named `UTC` found for enum `Tz` in the current scope
  --> C:\Users\Fred\.cargo\registry\src\index.crates.io-6f17d22bba15001f\chrono-tz-0.8.6\src\timezone_impl.rs:14:13
   |
14 |         Tz::UTC
   |             ^^^ variant or associated item not found in `Tz`
   |
  ::: C:\prj\target\release\build\chrono-tz-5b053da3462d6436\out/timezones.rs:13:1
   |
13 | pub enum Tz {
   | ----------- variant or associated item `UTC` not found for this enum

For more information about this error, try `rustc --explain E0599`.
error: could not compile `chrono-tz` (lib) due to 1 previous error
```

Why?

Is UTC stripped out too? Should we update the Readme?

--- a/bin/test-regex-filtering.sh
+++ b/bin/test-regex-filtering.sh
@@ -3,7 +3,7 @@
 set -euxo pipefail
 
 export RUST_BACKTRACE=1
-export CHRONO_TZ_TIMEZONE_FILTER='(Europe/London|GMT|UTC)'
+export CHRONO_TZ_TIMEZONE_FILTER='(Europe/London)'
 
 cd chrono-tz/tests/check-regex-filtering
 

--- a/chrono-tz-build/src/lib.rs
+++ b/chrono-tz-build/src/lib.rs
@@ -328,6 +328,8 @@ mod filter {
 }
 
 /// Module containing code supporting filter-by-regex feature
+///
+/// The "GMT" and "UTC" time zones are always included.
 #[cfg(feature = "filter-by-regex")]
 mod filter {
     use std::collections::HashSet;
@@ -390,13 +392,13 @@ mod filter {
     fn filter_timezone_table(table: &mut Table, filter_regex: Regex) {
         // Compute the transitive closure of things to keep.
         // Doing this, instead of just filtering `zonesets` and `links` by the
-        // regiex, helps to keep the `structure()` intact.
+        // regex, helps to keep the `structure()` intact.
         let mut keep = HashSet::new();
         for (k, v) in &table.links {
-            if filter_regex.is_match(k) {
+            if filter_regex.is_match(k) || k == "GMT" || k == "UTC" {
                 insert_keep_entry(&mut keep, k);
             }
-            if filter_regex.is_match(v) {
+            if filter_regex.is_match(v) || k == "GMT" || k == "UTC" {
                 insert_keep_entry(&mut keep, v);
             }
         }

